docs/en/query_language/select.md

```diff
@@ -382,9 +382,9 @@ The table names can be specified instead of `<left_subquery>` and `<right_subque
 - `FULL OUTER JOIN`
 - `CROSS JOIN`
 
-You may skip the `OUTER` keyword because it is implied by default.
+You may skip the `INNER` and `OUTER` keywords because they are implied by default.
```

Comment:
not exactly: if you skip all of them `INNER` is implied, but if you specify any of the alternatives to INNER (LEFT/RIGHT/FULL/etc.) `OUTER` is implied

I don't remember if we generate an error on `INNER OUTER JOIN`, but logically it's incorrect